### DISCUSSION TITLE
Logitech G Saitek Pro Flight Radio Panel

### DISCUSSION THREAD

**Discussion by TerminalR00T** [flybywiresim/aircraft#6246] [Help]
Hi All,

My apologies if this has already been asked.  I have the Logitech G Saitek Pro Flight Radio Panel and was wondering if anyone had got it working with FBW? It works on the normal sim but not with the FBW aircraft loaded.

Any comments or suggestions welcome.

**Comment by speedwaystar** [comment]
320simpilot (famous A32NX youtuber) uses it with the FBW in all his streams.

**Comment by speedwaystar** [comment]
i completely forgot (luckily he mentioned it in his most recent stream) that he had to bind AVIONICS MASTER ON to a key and press it before the panel would work properly.

**Comment by TerminalR00T** [comment]
Perfect..!! thanks that's done the job nicely.

![20211204_213236519_iOS](https://user-images.githubusercontent.com/95051716/144826630-8ff1c801-ff1d-4ade-8d0b-d6c1067bcf63.jpg)
